tests/functional/testUtapiClient.js

```diff
@@ -49,6 +106,53 @@ describe('Counters', () => {
                 next),
             next => utapiClient.pushMetricCreateBucket(reqUid, testBucket,
                 next),
-        ], () => _assertCounters(testBucket, done));
+        ], () => _assertCounters(testBucket, {
+            storageUtilized: 0,
+            numberOfObjects: 0,
+        }, done));
     });
+
+    it('should update counters to correct values < 0: putObject', done =>
+        _checkMissingOperations(_assertCounters, 'PutObject', counterKeyValues,
+            done));
+
+    it('should update counters to correct values < 0: copyObject', done =>
+        _checkMissingOperations(_assertCounters, 'CopyObject', counterKeyValues,
+            done));
+
+    it('should update counters to correct values < 0: completeMPU', done =>
+        _checkMissingOperations(_assertCounters, 'CompleteMultipartUpload',
+            counterKeyValues, done));
+
+    it('should update counters to correct values < 0: uploadPart', done =>
+        // Uploading a part does not increment numberOfObjects value.
+        _checkMissingOperations(_assertCounters, null,
```

Comment:
Thank you. I had an internal debate about this, so I will update as you suggest. :)